manticore/core/workspace.py

```diff
@@ -94,7 +94,7 @@ def load_value(self, key):
             return s.read()
 
     @contextmanager
-    def save_stream(self, key):
+    def save_stream(self, key, *rest, **kwargs):
```

Comment:
Nothing currently uses `*rest`, but it's used in e.g. https://github.com/trailofbits/manticore/blob/master/manticore/core/workspace.py#L440-L441